### DISCUSSION TITLE
Feature request: Show most active forks

### DISCUSSION THREAD

**Discussion by jarvisjohnson** [community/community#47697] [Repositories]
### Select Topic Area

Product Feedback

### Body

For unmaintained projects, it is hard to tell which forks are still active.

It would be great to see a metric about recent activity in the forks to tell which ones are maintained.

Currently the forks page appears to be a static list, not clear what it is sorted by.  Perhaps most active should show up on top?

Thanks for considering this!

Note: Saw a better request for this, happy for this to be archived but I think it is an important feature so want to resurface this: https://github.com/orgs/community/discussions/34166

**Comment by Lxstr** [comment]
We absolutely need github to act on unmaintained repos, they are a huge problem and I don't understand why github wont even try to address them.
